docs/cli/kops_get.md

```diff
@@ -21,9 +21,12 @@ Display one or many resources.
   # Get a cluster
   kops get cluster k8s-cluster.example.com
   
-  # Get a cluster YAML cluster spec
+  # Get a cluster YAML configuration
```

Comment:
The problem here is that the command `kops get cluster -o yaml`
allows to get the _desired_ configuration (the specification) AND also allows to get the _actual_ configuration (`cluster.spec`, which I'm not 100% sure this is the actual configuration), with `-o yaml --full`.

To be clearer, I updated "cluster spec" to "cluster desired configuration" and added another example, "Get a cluster YAML cluster actual configuration".
How is that?